packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1059,23 +1084,25 @@ See [Okta Expression Language in Identity Engine](/docs/reference/okta-expressio
 
 ## Type-Specific Policy data structures
 
-## Okta Sign On Policy
+## Global Session Policy
 
-Okta Sign On Policy controls the manner in which a user is allowed to sign on to Okta, including whether they are challenged for multifactor authentication (MFA) and how long they are allowed to remain signed in before re-authenticating.
+> **Note:** The Okta Sign On Policy name has changed to Global Session Policy. The policy type of `OKTA_SIGN_ON` remains unchanged.
```

Comment:
We didn't rename this page on Classic did we? I wonder if we need to specify scope of the change to OIE